libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -460,6 +586,9 @@ bool AC_Fence::sys_status_enabled() const
     if (_action == AC_FENCE_ACTION_REPORT_ONLY) {
         return false;
     }
+    if (!_enabled) {
```

Comment:
OK, you might have some fights on your hands here.  I'm not *that* fussed at this point, but changing behaviour like this could get interesting :-)

Reply:
Figured that might be the case. Who's best to give feedback here?

What defines enabled?
What defines present?
What defines healthy?

Reply:
I've reverted this back to not include the enabled check here. I'm not actually sure where this gets rendered after it's reported out. 

My assumptions were:
 - `present` is `true` if a fence (inclusion/exclusion circle/poly) is present.
 - `enabled` is true when the fence is enabled.
 - `healthy` is true if not breached.

If this flag is being used to let the user know if the fence is enabled, how do we accurately do that without looking at the enabled flag itself? I may have turned a fence off with a switch, but this flag doesn't know that.